tests/common/utilities.py

```diff
@@ -531,3 +532,13 @@ def check_qos_db_fv_reference_with_table(duthost):
         logger.info("DUT release {} exits in release list {}, QOS db field value refered to table names".format(duthost.sonic_release, ", ".join(release_list)))
         return True
     return False
+
+
+def str2bool(str):
+    """
+    This is used as a type when add option for pytest
+    :param str: The input string value
+    :return: False if value is 0 or false, else True
+    """
+    return str.lower() not in ["0", "false"]
```

Comment:
Also add support to `no`?

Reply:
Thank you for the comments,  a new commit has been submitted to support "no"